packages/connector-oauth2/src/index.ts

```diff
@@ -33,9 +32,14 @@ const getAuthorizationUri =
     validateConfig<OauthConfig>(config, oauthConfigGuard);
     const parsedConfig = oauthConfigGuard.parse(config);
 
-    const parameterObject = snakecaseKeys(
-      omit(parsedConfig, ...oauthConfigGlobalKeys, 'clientSecret', 'grantType')
-    );
+    const { customConfig, ...rest } = parsedConfig;
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const parameterObject = snakecaseKeys({
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-call
+      ...pick(rest, 'responseType', 'clientId', 'scope'),
```

Comment:
why the pick method returns any type? This looks odd. Did you pick the latest '@silverhand/essentials' package?

Reply:
Per offline discussion, switch to `pick` from `omit`. Since this change comes with the update of configuration guards, we can do this change before we publish these two standard connectors. As long as connectors are migrated to ESM, we can update to the latest version of '@silverhand/essentials', and dependencies on `lodash` will be automatically removed.